fury/tests/test_window.py

```diff
@@ -332,3 +333,95 @@ def test_content(filename='fury.png', colors_found=(True, True)):
 
             assert_less_equal(arr.shape[0], 5000)
             assert_less_equal(arr.shape[1], 5000)
+
+
+def test_opengl_state_simple():
+    for gl_state in [
+        window.gl_reset_blend, window.gl_enable_depth,
+        window.gl_disable_depth, window.gl_enable_blend,
+        window.gl_disable_blend,
+        window.gl_set_additive_blending,
+        window.gl_set_normal_blending,
+        window.gl_set_multiplicative_blending,
+        window.gl_set_subtractive_blending,
+        window.gl_set_additive_blending_white_background
+    ]:
+        scene = window.Scene()
+        centers = np.array([
+            [0, 0, 0],
+            [-.1, 0, 0],
+            [.1, 0, 0]
+        ])
+        colors = np.array([
+            [1, 0, 0],
+            [0, 1, 0],
+            [0, 0, 1]
+        ])
+
+        actors = actor.markers(
+            centers,
+            marker='s',
+            colors=colors,
+            marker_opacity=.5,
+            scales=.2,
+        )
+        showm = window.ShowManager(
+            scene,
+            size=(900, 768), reset_camera=False,
+            order_transparent=False)
+
+        scene.add(actors)
+        # single effect
+        shaders.shader_apply_effects(
+            showm.window, actors,
+            effects=gl_state)
+        showm.render()
+
+
+def test_opengl_state_add_remove_and_check():
+    scene = window.Scene()
+    centers = np.array([
+        [0, 0, 0],
+        [-.1, 0, 0],
+        [.1, 0, 0]
+    ])
+    colors = np.array([
+        [1, 0, 0],
+        [0, 1, 0],
+        [0, 0, 1]
+    ])
+
+    actor_no_depth_test = actor.markers(
+        centers,
+        marker='s',
+        colors=colors,
+        marker_opacity=.5,
+        scales=.2,
+    )
+    showm = window.ShowManager(
+        scene,
+        size=(900, 768), reset_camera=False,
+        order_transparent=False)
+
+    scene.add(actor_no_depth_test)
+
+    showm.render()
+    state = window.gl_get_current_state(showm.window.GetState())
+    before_depth_test = state['GL_DEPTH_TEST']
+    assert before_depth_test
```

Comment:
We do not use ``assert`` statements like that in FURY tests. Please use numpy testing as we use in the other tests. Numpy testing is more explicit and therefore preferable. Also makes it easy to test arrays etc.